pkg/volume/util/util.go

```diff
@@ -108,6 +108,13 @@ func SetReady(dir string) {
 	file.Close()
 }
 
+// CleanReady remove the 'ready' file
+func CleanReady(dir string) {
+	if err := os.RemoveAll(dir); err != nil && !os.IsNotExist(err) {
```

Comment:
Instead of removing the entire dir, should we be removing only the `ready` file?

Reply:
readyDir should be empty, I think. But to be safe, maybe we call os.Remove() and throw error if it is not empty.

Reply:
actually getMetaDir contains one file named "ready", there should no other files. I think RemoveAll is fine.